editions/tw5.com/tiddlers/variables/modifier Variable.tid

```diff
@@ -4,7 +4,7 @@ tags: Variables [[Core Variables]]
 title: modifier Variable
 type: text/vnd.tiddlywiki
 
-Within the ''action'' string of the DroppableWidget, the ''action'' string of the ButtonWidget and the ''action'' string of the LinkCatcherWidget and the EventCatcherWidget, the <<.def modifier>> [[variable|Variables]] contains the modifier key(s) held during the drag-process.
+Within the ''action'' string of the DroppableWidget, the ''startactions'' and ''endactions'' of the DraggableWidget, the ''action'' string of the ButtonWidget and the ''action'' string of the LinkCatcherWidget and the EventCatcherWidget, the <<.def modifier>> [[variable|Variables]] contains the modifier key(s) held during the drag-process / click / event.
```

Comment:
The docs were already in poor English. Better to say:

```
...held during the drag, click or other event.
```

Reply:
Thanks @Jermolene , I've updated the docs